### ISSUE TITLE
Audio Station on Synology DSM 6 doesn't work

### ISSUE THREAD

**Issue by alexteg** [borismus/keysocket#173]
keysocket doesn't seem to inject itself into pages running Synology DSM 6. The manifest currently only captures pages containing `"*://*/webman/*"`, while DSM no longer has `webman` in the URL and redirects any access to that URL to just `index.cgi`, which also may be omitted. The DSM can now be loaded with the bare domain + port that it resides on, such as `https://foobar.synology.me:5001/` without any additional path, which considering the domain & port is adjustable makes it difficult to capture all possible URL combinations.

**Comment by eguilder** [comment]
Just make it capture pages on ports 5000/5001 that should be good enough

**Comment by feedbee** [comment]
Is it still actual problem? Can we catch `https://*.synology.me/*`?

**Comment by feedbee** [comment]
Demo version of DSM is running at `https://***.demo.synology.com:5001/`. So, to catch `https://*.synology.me/*` is not a solution. While we know nothing about domain, we can't detect the site.

There are different variants to handle this problem, but all of them have weaknesses:
1) Check all sites for some condition to detect DSM. - Resources waste for all extension users on all sites for this check.
2) Give extension user a tool to set custom domain for plugin. - We need to build options window, which is absent currently.
3) Give extension user a tool to set custom handler for custom domain: more general solution, but still - We need the window.

I think we have to leave this issue opened as feature request for custom configuration.

**Comment by wzooff** [comment]
One more [solution](https://www.synology.com/en-us/knowledgebase/DSM/help/AudioStation/AudioStation2_desc). Works for me, when i've set "audiostation" as an alias 
<img width="419" alt="audiostation" src="https://cloud.githubusercontent.com/assets/1210371/26831530/3f3e3b34-4ad5-11e7-8d15-0ebeb2ad6c61.png">
And now i can open Audio Station as separate tab in browser if type in url field https://mysyno.com/audiostation/
Anyway its compatibility break, we cant use "webman" coz it for system use only

**Comment by feedbee** [comment]
Thank you, @wzooff. But It's still a workaround, not a reliable and straightforward solution.

**Comment by wzooff** [comment]
Yep, but its mostly synology owners problem, than keysocket. In keysocket readme we see "when you're listening to music on various streaming websites" and keysocket has list of supported websites (without dsm v6). It's by design. Synology NAS is not a streaming website. For me its not a problem to configure alias :) maybe we need amend documentation or create FAQ/remark for DSM6

Or better to extend feature request and create configuration page (solution #3)

**Comment by eguilder** [comment]
I tried wzooff's solution and it didn't work:

1. have audio station running on: http://192.168.192.69:5000/audiostation/
2. I edited the manifest.json to have: {
      "js": [ "shared.js", "keysocket-synology.js" ],
      "matches": [ "*://*/webman/*", "*://*/audiostation/" ]
   }

But the extension doesn't get enabled when I am on the audio station URL. What could be wrong?
